group_vars/pdc_discovery/staging.yml

```diff
@@ -15,6 +15,11 @@ rails_app_env: "staging"
 pdc_discovery_host_name: 'pdc-discovery-staging.princeton.edu'
 pdc_discovery_honeybadger_key: '{{vault_pdc_discovery_honeybadger_key}}'
 
+install_mailcatcher: true
+mailcatcher_user: "pulsys"
+mailcatcher_group: "pulsys"
+mailcatcher_install_location: "/usr/local/lib/ruby/gems/3.0.0/gems/mailcatcher-0.8.2/bin/mailcatcher"
```

Comment:
How do I find the "right" folder for the gem in this server? In `pdc-discovery-staging2` for example the gems are installed under `/usr/local/lib/ruby/gems/3.3.0` (not under 3.0.0)